knative-operator/deploy/resources/dashboards/serving/grafana-dash-knative-serving-service-queue-proxy.yaml

```diff
@@ -0,0 +1,638 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: grafana-dash-knative-serving-service-queue-proxy
```

Comment:
This ConfigMap name should be consistent with the ConfigMap (`grafana-dashboard-definition-knative-serving-resources`):
https://github.com/openshift-knative/serverless-operator/blob/fe8d29ddb5d34cd47c352d476f169dcc38c5aa72/knative-operator/deploy/resources/dashboards/serving/grafana-dash-knative-serving-resources.yaml#L18

Probably `grafana-dashboard-definition-queue-proxy` or something?

Reply:
Changed it to: `grafana-dash-definition-queue-proxy`.